### PR TITLE
user0-general pull request

### PR DESCRIPTION
**Pull request by LucyMerlier**
Création du fichier base.html.twig, et mise en place de la base de données (création de db.sql)

### CODE REVIEW

**Review by Mgaren — DISMISSED** [review]

File: src/View/base.html.twig [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    <header>
+        <h1>Quizzy</h1>
+    </header>
+
+    {% block content %}
+    {% endblock %}
+
+    {% block scripts %}
+    {% endblock %}
```
Comment (RomainClair) [inline src/View/base.html.twig]: If the idea of the script block is to add javascripts, it probably should be at the end of the body : after the footer

File: src/View/base.html.twig [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    <meta charset="UTF-8">
+    <title>
+        {% block title %}
+        Quizzy
+        {% endblock %}
+    </title>
+
+    {% block stylesheets %}
+    <link rel = "stylesheet" href = "mainStyle.css">
```
Comment (RomainClair) [inline src/View/base.html.twig]: /public/mainStyle.css doesn't exists
There's already a /public/assets/css directory where the simple-mvc framework seems to store its css files.

**Review by RomainClair — CHANGES_REQUESTED** [review]
There's a new db.sql script. You should probably update the README.me (Or create an INSTALL.md) to explain what to do with this script.
Should i import simple-mvc.sql then db.sql or only one or... ?

It seems a bit strange to have this amount of blank lines in the base.html.twig

Why is there a base and a layout html.twig files because both seems to be a whole page template.

File: src/View/base.html.twig (hunk 1 above)
Comment (LucyMerlier) [reply src/View/base.html.twig]: updated

File: src/View/base.html.twig (hunk 2 above)
Comment (LucyMerlier) [reply src/View/base.html.twig]: updated

**Review by Mgaren — DISMISSED** [review]

**Review by RomainClair — CHANGES_REQUESTED** [review]
In general, you'll have to choose the most adequate language for your project and stick to it for all your docs, comments, etc.

In this project, your comments must be in english.
But it's a really good idea to comment your .sql file =)

Some foreign keys seems to be missing (theme <-> question, theme <-> author)

**Review by Mgaren — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**